siswrap/wrapper_services.py

```diff
@@ -244,12 +246,14 @@ def new_wrapper(wrapper_type, runfolder, configuration_svc):
             return AeacusStatsWrapper(runfolder, configuration_svc)
         elif wrapper_type == Wrapper.AEACUS_REPORTS_TYPE:
             return AeacusReportsWrapper(runfolder, configuration_svc)
+        elif wrapper_type == Wrapper.CHECK_INDICES_TYPE:
+            return CheckIndicesWrapper(runfolder, configuration_svc)
         else:
             raise RuntimeError("Unknown wrapper runner requested: {0}".
                                format(wrapper_type))
 
 
-class AeacusBaseWrapper(Wrapper):
+class BaseWrapperWithoutEmail(Wrapper):
     """
     Base wrapper for the Aeacus commands
```

Comment:
Perhaps you could update this docstring, as well for `get_exec_string()`, `AeacusStatsWrapper()` and `AeacusReportsWrapper()`.